netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -434,6 +524,40 @@ public static ProtocolNegotiator tls(SslContext sslContext) {
     return tls(sslContext, null);
   }
 
+  public static ProtocolNegotiator.ClientFactory tlsClientFactory() {
```

Comment:
There's only one reference to this function. Can that be replaced with a call to `tlsClientFactory(null)` ?